### ISSUE TITLE
 Can't find the button in add files

### ISSUE THREAD

**Issue by DynamticS** [vantezzen/zotero-better-bibtex-to-overleaf#2]
Thank you for uploading the plugin, it greatly facilitates the synchronization of documents between Zotero and Overleaf, but I don't know if it's because of Overleaf's update, lately I can't find the button in add files anymore.

**Comment by angelakmiller** [comment]
+1

**Comment by vantezzen** [comment]
This was fixed by #1 but it looks like I forgot to release a new version. Just pushed the new version to Chrome and Firefox - should be available as soon as their teams approve the new version!

As a temporary fix while the function isn't working, you can manually create a ".bib" or ".bibtex" file and paste this into it:

```
# BETTER BIBTEX
# http://127.0.0.1:23119/better-bibtex/export/collection?/1/YOUR_URL.bibtex
``` 

replacing the second line with your individual URL. In the editor, switch to any other file, then back to the new .bib file so the extension detects the header.

**Comment by DynamticS** [comment]
Thank you for updating, it seems that it has been resolved ：）
